bms.py

```diff
@@ -0,0 +1,123 @@
+from math import isnan
+from bmsConstants import bms_param_thresholds 
+from bmsConstants import bms_multilingual_logs
+from bmsConstants import bms_multilingual_logs_status_heading
+from bmsConstants import language_description
+from bmsConstants import bms_sensor_temperature_unit
+from bmsConstants import bms_sensor_temperature_log
+def printglobalparamLog(log_string, mode):

```

Comment:
What is the meaning of mode? What would be an intuitive name? And any possibility to test it?

Reply:
mode here is just stating that a log for the global parameter should be logged in console or not, also it is just differentiating the purpose of testing and actual running of the method.
if mode is set as false it is just bypassing the logging part that is why i kept false while testing the same.

![image](https://user-images.githubusercontent.com/15873999/110743890-d1016b80-825e-11eb-955f-ffca65400e5a.png)

of course we can have a change the name to something else such as 'log_mode' default it True for testing purpose can be passed false to it